src/fparser/two/tests/fortran2003/test_usestmt_r1109.py

```diff
@@ -281,12 +369,12 @@ def test_use_internal_error2(f2003_create):
         ast.items = (None, None, content, None, None)
         with pytest.raises(InternalError) as excinfo:
             str(ast)
-        assert ("entry 2 should be a module name but it is " "empty") in str(
+        assert "entry 2 should be a module name but it is " "empty" in str(
```

Comment:
Why is there a `" "` between 'is' an 'empty'? as black rearraged this incorrecty?

It happens in a few places:
src/fparser/two/tests/fortran2003/test_usestmt_r1109.py
372:        assert "entry 2 should be a module name but it is " "empty" in str(

src/fparser/two/tests/fortran2003/test_cray_pointer_decl.py
118:        assert ("'Items' entry 0 should be a pointer name but it is " "empty") in str(

src/fparser/two/tests/fortran2003/test_position_edit_desc_r1013.py
140:        "items[1] in Class Position_Edit_Desc method tostr() is " "empty or None"

src/fparser/two/tests/fortran2003/test_hollerith_item.py
120:        assert ("entry 0 should be a valid Hollerith string but it is " "empty") in str(

Reply:
Weird. I think that must have been Black. I've fixed them now.